artifacts/config_files/sfall-mods.ini

```diff
@@ -4,8 +4,14 @@
 ;  42 - SHIFT key
 Key=42
 
-; Set to 1 also highlight containers
-Containers=1
+; Set to 1 to also highlight containers
+Containers=0
+
+; Set to 1 to also highlight corpses
+Corpses=0
```

Comment:
I think the most logical default from user point of view is to highlight everything with items in it (items, containers AND corpses), maybe even everything interactable (that has "use" icon - not sure how to script it yet). I agree about LOS check disabled by default.